lib/miq_automation_engine/service_models/miq_ae_service_service_template_transformation_plan_task.rb

```diff
@@ -27,5 +27,10 @@ def transformation_destination(source_obj)
         wrap_results(@object.transformation_destination(source_obj.object_class.find(source_obj.id)))
       end
     end
+
+    def conversion_host=(conversion_host)
+      raise ArgumentError, "conversion_host must be nil or a MiqAeServiceConversionHost" unless conversion_host.nil? || conversion_host.kind_of?(MiqAeMethodService::MiqAeServiceConversionHost)
+      ar_method { wrap_results(@object.conversion_host = ConversionHost.find(conversion_host.id)) }
```

Comment:
Now that we are back to supporting `nil` as a potential host parameter the `conversion_host.id` call will fail with `undefined method 'id' for nil:NilClass`

Which highlights the need for tests on the PR.